src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const humans = document.querySelector('ul');
+
+function clearString(string) {
+  return string.slice(1).split(',').join('');
+}
+
+function sortList(list) {
+  const listChildrenArray = [...list.children];
+
+  return listChildrenArray.sort((prChild, curChild) => {
+    const prChildSalary = clearString(prChild.dataset.salary);
+    const curChildSalary = clearString(curChild.dataset.salary);
+
+    return curChildSalary - prChildSalary;
+  });
+}
+
+const sortedHumanList = sortList(humans);
+
+humans.append(...sortedHumanList);
+
+function getEmployees(list) {
+  const humanObjArr = [];
+  const childrenArr = [...list.children];
+
+  for (let i = 0; i < childrenArr.length; i++) {
+    humanObjArr.push(
+      {
+        name: childrenArr[i].innerText,
+        position: childrenArr[i].dataset.position,
+        salary: childrenArr[i].dataset.salary,
+        age: childrenArr[i].dataset.age,
+      }
+    );
+  }
+
+  return humanObjArr;
+}
```

Comment:
You cold just return `[...list.children].map(...)` instead of creating 'humanObjArr' variable and 'for' loop